packages/javascript/bh-shared-ui/src/components/HelpTexts/ADCSESC9b/General.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { FC } from 'react';
+import { groupSpecialFormat } from '../utils';
+import { EdgeInfoProps } from '../index';
+import { Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme) => ({
+    containsCodeEl: {
+        '& code': {
+            backgroundColor: 'darkgrey',
+            padding: '2px .5ch',
+            fontWeight: 'normal',
+            fontSize: '.875em',
+            borderRadius: '3px',
+            display: 'inline',
+
+            overflowWrap: 'break-word',
+            whiteSpace: 'pre-wrap',
+        },
+    },
+}));
```

Comment:
I know this is a pattern we are already using in other help texts, and fixing them all may be beyond the scope of this ticket, but it would be good to pull these styles into a separate file to keep code formatting consistent -- from what I can see they are the same everywhere currently.

Reply:
Good point! The `<Links />` component is similarly repetitive so might as well pull that out and refactor a couple of these as well!